IPython/utils/process.py

```diff
@@ -41,6 +41,9 @@ def find_cmd(cmd):
         from IPython.utils.process import pycmd2argv
         argv = pycmd2argv(get_ipython_module_path('IPython.terminal.ipapp'))
 
+    Note, The code for pycmd2argv has been removed now as it was not used
+    anywhere. get_ipython_module_path should give us that same result.
```

Comment:
Hi @takluyver , thank you for reviewing the pull request. I do have one doubt.

What I understood from pycmd2argv and find_cmd functions, the need is to give the actual path where a module resides. And the previous author had mentioned that the pycmd2argv was a function used for IPython modules.

Can I replace the above set of lines with this example ?

        import sys 
        argv = [sys.executable, get_ipython_module_path('IPython.terminal.ipapp')]

Please let me know if you are suggesting something on other lines.